specification/semantic_conventions.md

```diff
@@ -99,7 +99,16 @@ For each `InstrumentationLibraryLogs` instance:
 - `name` - MUST be set to `otel.profiling`
 - `version` - MUST be set to `0.1.0`
 
-The resource field in `ResourceLogs` MUST contain the [`telemetry.sdk.language`](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/resource/semantic_conventions#telemetry-sdk) attribute.
+The resource field in `ResourceLogs`:
+
+- MUST contain following attributes:
+  - [`telemetry.sdk.language`](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/resource/semantic_conventions#telemetry-sdk)
+  - `service.name`
```

Comment:
Why not to add links to all of these semantic attributes?

Reply:
added in https://github.com/signalfx/gdi-specification/pull/206/commits/2109247be55348a9edfdfa3212b27ec9706f5025